### DISCUSSION TITLE
user groups for permission

### DISCUSSION THREAD

**Discussion by bVIIw** [plausible/analytics#1956] [Anything Else]
We would very much appreciate it if user groups could be introduced which can then be authorized for the sites. 

In that manner that you can authorize a group for one or more sites and then put a new user only once in this group and he has rights on all sites on which the group is authorized. 

As it is currently, this is very inconvenient if you have to manually authorize a new employee/customer each time on several sites.

Thanks in advance!

**Comment by metmarkosaric** [comment]
thanks for the feedback @bVIIw, it makes sense! we have a thread on this topic already here: https://github.com/plausible/analytics/discussions/1350

**Reply by bVIIw** [reply]
thanks for your reply, haven't seen the referenced case - i only searched for keywords like groups and rbac and so on ;)

**Reply by metmarkosaric** [reply]
no worries! your post will help others find the right thread in the future :)
